src/wizard/obtain_source.jl

```diff
@@ -51,8 +51,13 @@ end
 function clone(url, source_path)
     p = Progress(0, 1, "Cloning: ")
     @compat_gc_preserve p begin
-        callbacks = LibGit2.RemoteCallbacks(transfer_progress=cfunction(transfer_progress, Cint, Tuple{Ptr{GitTransferProgress}, Any}),
-            payload = pointer_from_objref(p))
+        @static if VERSION >= v"0.7-"
+            callbacks = LibGit2.RemoteCallbacks(transfer_progress=@cfunction(transfer_progress, Cint, (Ptr{GitTransferProgress}, Any)),
```

Comment:
You can just use `@cfunction` from Compat.

Reply:
Femtocleaner will remove the version check automatically, and it's required for the next line anyway.